presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMetadataUpdater.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.spi.ConnectorMetadataUpdateHandle;
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.facebook.presto.hive.TestHiveMetadataUpdateHandle.TEST_PARTITION_NAME;
+import static com.facebook.presto.hive.TestHiveMetadataUpdateHandle.TEST_SCHEMA;
+import static com.facebook.presto.hive.TestHiveMetadataUpdateHandle.TEST_TABLE;
+import static org.testng.Assert.assertEquals;
+
+public class TestHiveMetadataUpdater
```

Comment:
Shall we have more multi-threading tests for this class?

Reply:
As per our new design , we have one `HiveMetadataUpdater` instance per each task. On other hand we need multi-threaded tests for `HiveFileRenamer` as we have a singleton on coordinator for all tasks. So added multi-threaded tests for `HiveFileRenamer`